src/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN apk add --no-cache \
     psmisc \
     shadow \
     sudo \
+    tini \
```

Comment:
Didn't you want to save 2 MB?...

Reply:
The smirking emoji was supposed to be representative of my not entire serious nature when saying this.